### PR TITLE
DOCS-3796: Update payment components and tokenization

### PR DESCRIPTION
**Pull request by Rendani-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

**Review by Daniel-MultiSafepay — APPROVED** [review]

File: content/integrations/prestashop-1-6.md [hunk 1]
```diff
@@ -130,16 +130,26 @@ The plugin supports [payment components](/docs/payment-components/), which:
 - Encrypt customer payment details for secure processing.
 - Shift responsibility for [PCI DSS compliance](/docs/pci-dss/) to MultiSafepay.
 
+<details id="supported-payment-methods">
+<summary>Supported payment methods</summary>
+<br>
+
+- Cards: Amex, Maestro, Mastercard, and Visa.
```
Comment (inesmayerMSP) [inline content/integrations/prestashop-1-6.md:137]: no full stop

File: content/integrations/prestashop-1-6.md [hunk 2]
```diff
@@ -130,16 +130,26 @@ The plugin supports [payment components](/docs/payment-components/), which:
...
 
-If you're new to accepting credit card payments, email a request to activate them to <sales@multisafepay.com>
-
 1. Sign in to your PrestaShop 1.6 backend.
-2. Go to **Modules and services** > **Credit cards**.
-3. Slide the **Use Payment component** radio button to **Yes**.
-4. Click **Save**.
+2. Go to **Modules and services**
+3. Select the relevant payment methods.
+4. Select the **Use Payment component** radio button to **Yes**.
```
Comment (inesmayerMSP) [inline content/integrations/prestashop-1-6.md:149]: ?

File: content/integrations/prestashop-1-6.md [hunk 3]
```diff
@@ -130,16 +130,26 @@ The plugin supports [payment components](/docs/payment-components/), which:
...
+5. Click **Save**.
+
+💬 Support: If you're new to accepting credit card payments, email a request to activate them to <sales@multisafepay.com>
```
Comment (inesmayerMSP) [inline content/integrations/prestashop-1-6.md:152]: review

File: content/integrations/prestashop-1-6.md [hunk 4]
```diff
@@ -156,10 +166,11 @@ Payment links generated manually in your MultiSafepay dashboard don't automatica
...
 
 1. Sign in to your PrestaShop 1.6 backend. 
-2. Go to **Modules and services** > **Modules and services** > **Credit cards**.
-3. Click **Configure**.
-4. Set the **Tokenization** field to **YES**.
-5. Click **Save**.
+2. Go to **Modules and services** > **Modules and services** 
+3. Select either the bundled credit cards <<glossary:gateway>>, **or** select specific credit cards.
+4. Click **Configure**.
+5. Set the **Tokenization** field to **YES**.
```
Comment (inesmayerMSP) [inline content/integrations/prestashop-1-6.md:172]: unify with above text: caps or not

File: content/integrations/prestashop-1-7.md [hunk 5]
```diff
@@ -143,16 +143,26 @@ The plugin supports [payment components](/docs/payment-components/), which:
 - Encrypt customer payment details for secure processing.
 - Shift responsibility for [PCI DSS compliance](/docs/pci-dss/) to MultiSafepay.
 
+<details id="supported-payment-methods">
+<summary>Supported payment methods</summary>
+<br>
+
+- Cards: Amex, Maestro, Mastercard, and Visa.
```
Comment (inesmayerMSP) [inline content/integrations/prestashop-1-7.md:150]: full stop

File: content/integrations/woocommerce.md [hunk 6]
```diff
@@ -188,18 +188,26 @@ The plugin supports [payment components](/docs/payment-components/), which:
...
 
-If you're new to accepting credit card payments, email a request to activate them to <sales@multisafepay.com>
-
 1. Sign in to your Wordpress backend.
-2. Go to **WooCommerce** > **MultiSafepay settings** > **Payment methods** > **Credit card**, and then click **Manage**.
-3. Select the **Payment components** checkbox.
-4. Click **Save changes**.
+2. Go to **WooCommerce** > **MultiSafepay settings** > **Payment methods** > 
+3. Select the relevant payment methods, and click **Manage**.
+4. Select the **Payment components** checkbox.
```
Comment (inesmayerMSP) [inline content/integrations/woocommerce.md:207]: verify. Select if more options

**Review by inesmayerMSP — CHANGES_REQUESTED** [review]

**Pull request merged**